telemetry/specs/INT.mdk

```diff
@@ -1596,15 +2034,62 @@ INT Metadata Header and Metadata Stack:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-## Example with INT over Geneve
+## Example with INT-MX over VXLAN GPE
+
+We now consider a scenario where Host1 and Host2 use VXLAN encapsulation.
+Host1 acts as VXLAN tunnel endpoint and INT source, inserts VXLAN and INT-MX
+header with instruction bits corresponding to the network state to be
+reported at intermediate switches. In this example, Host1 itself does
+not send any metadata to the monitoring system. Intermediate switches parse 
+through VXLAN header and and processes the INT-MX header and sends the INT 
```

Comment:
s/and and processes/, process
s/sends/send

Reply:
Done